src/components/App/index.js

```diff
@@ -155,11 +156,32 @@ export class App extends Component {
 		I18n.on('change', this.handleLanguageChange);
 	}
 
+	location = async() => {
+		const { ip } = await axios.get(`${ 'https://cors-anywhere.herokuapp.com/' }https://api.ipify.org?format=json`)
+			.then((res) => res.data)
+			.catch((err) => err);
+
+		const location = await axios.get(`${ 'https://cors-anywhere.herokuapp.com/' }https://geoip-db.com/json/${ ip }`)
+			.then((res) => res.data)
+			.catch((err) => err);
+
+		const token = getToken();
+		return {
+			location,
+			token,
+		};
+	}
```

Comment:
I've some suggestions:

1. Check if GDPR protection is enabled and the user has approved its data collection;
2. Use the own browser's geolocation API before using an external service;
3. If the geolocation API fails, request to `geoip-db.com` as a fallback;
4. Verify the limits for commercial use off `geoip-db.com` -- despite Rocket.Chat and this client being MIT; 
5. The `fetch()` API has some options to properly do CORS requests;
6. The ipify functionally can be implemented on Rocket.Chat's Livechat API, decreasing the number of requests needed.

Reply:
Thanks for the review, will update it accordingly